src/config/app.config.test.ts

```diff
@@ -79,124 +77,106 @@ describe('appConfig', () => {
     });
   });
 
-  describe('parsing REDACT_LOGS', () => {
-    it('sets redactLogs to true if REDACT_LOGS is true', () => {
-      replaceEnvironmentVariables({
-        REDACT_LOGS: 'true',
-      });
-
-      const config = appConfig();
-
-      expect(config.redactLogs).toBe(true);
-    });
-
-    it('sets redactLogs to false if REDACT_LOGS is false', () => {
-      replaceEnvironmentVariables({
-        REDACT_LOGS: 'false',
-      });
-
-      const config = appConfig();
-
-      expect(config.redactLogs).toBe(false);
-    });
-
-    it('sets redactLogs to true if REDACT_LOGS is not specified', () => {
+  describe('parsing HTTP_VERSIONING_ENABLE', () => {
```

Comment:
We have this as separate tests as it is a nested object and time constraints means we are not able to rework the existing helper implementation